lib/restrictions.js

```diff
@@ -31,9 +31,9 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
 /**
  * Inserts a restriction.
  *
- * @param {object} options - Options to use.
- * @param {object} options.restriction - An application-specific string that
- *   unambiguously identifies a particular entity to the application.
+ * @param {*} options - Options to use.
```

Comment:
I couldn't figure out a way to write jsdocs for three layer nested objects. This was the only way I could find to make the lint pass.

Reply:
this works for me:

```js
@param {object<object>} [options = {}] - Options to use.
```

however this also works

```js
@param {object<string>} [options = {}] - Options to use.
@param {object<Array>} [options = {}] - Options to use.
@param {object<number>} [options = {}] - Options to use.
```

so it appears the jsdoc plugin (which has caused issues before) is kind of broken here.